client/state/sharing/keyring/actions.js

```diff
@@ -12,9 +17,11 @@ import {
 /**
  * Triggers a network request for a user's connected services.
  *
+ * @param {Function} successCallback Function that is called after a successful
+ *                                   request. Default: _.identity.
  * @return {Function} Action thunk
  */
-export function requestKeyringConnections() {
+export function requestKeyringConnections( successCallback = identity ) {
```

Comment:
That makes a lot more sense. Reverted in 447c87c9eb1ad2c0354e07867a4c395c26dffd0d